presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorMetadata.java

```diff
@@ -167,6 +170,7 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         if (columns.isEmpty()) {
             throw new PrestoException(RAPTOR_ERROR, "Table does not have any columns: " + tableName);
         }
+        columns.add(hiddenColumn(SHARD_UUID_COLUMN_NAME, VARCHAR));
```

Comment:
The list from `toList()` is not guaranteed to be mutable. Use `toCollection(ArrayList::new)`